### PR TITLE
Fix/tests

### PR DESCRIPTION
**Pull request by sk91**
related to #368

### CODE REVIEW

File: src/main/db/test-db.js [hunk 1]
```diff
@@ -1,33 +1,56 @@
...
-		await db.knex.raw('select 1+1 as result');
-		return false;
-	} catch (err) {
-		await db.knex.client.initializePool(db.config);
-		return true;
+export class TestDb {
+	static async init() {
+		try {
+			this.knex = Knex(config);
+			this.oldKnex = Model.knex();
```
Comment (dfcowell) [inline src/main/db/test-db.js]: Why are we keeping a reference to the old instance of knex?

File: src/main/db/test-db.js [hunk 2]
```diff
@@ -1,33 +1,56 @@
...
+		}
 	}
-	await init();
-};
-
-export { init, reset };
+	static async reset() {
+		try {
+			await this.knex.destroy();
+			Model.knex(this.oldKnex);
```
Comment (dfcowell) [inline src/main/db/test-db.js]: I don't understand what this is doing.

Comment (sk91) [reply src/main/db/test-db.js]: @dfcowell that's a result of a very long fight with db connection errors in tests. This one is a leftover, will remove

**Comment by sk91** [comment]
@dfcowell fixed

File: package.json [hunk 3]
```diff
@@ -32,12 +32,14 @@
   "main": "./src/main/main.js",
   "scripts": {
     "dev": "gulp && electron-webpack dev",
-    "postinstall": "electron-builder install-app-deps buildDependenciesFromSource",
+    "install-app-deps": "electron-builder install-app-deps buildDependenciesFromSource",
+	"install-all": "yarn && yarn install-app-deps",
```
Comment (rodrigopavezi) [inline package.json]: @sk91 curious, why did you remove `postinstall`?  Cheers

**Comment by rodrigopavezi** [comment]
@sk91 looks really good just a comment about the install-all. cheers

File: package.json (hunk 3 above)
Comment (sk91) [reply package.json]: @rodrigopavezi it takes a lot of time becouse it rebuilds everything from source. For unit tests it is not needed

Comment (rodrigopavezi) [reply package.json]: @sk91 makes sense. cheers
